index.js

```diff
@@ -4,8 +4,8 @@ module.exports = () => {
   const router = new Router();
 
   function createHandler(method) {
-    return function handler(path, fun) {
-      return router[method](path, (req, res, next) => {
+    return function handler(path, ...rest) {
```

Comment:
Depends on which version of node we're targeting (http://node.green/#rest-parameters).